components/_patterns/04-templates/example-page/example-page.twig

```diff
@@ -0,0 +1,94 @@
+<div class="example-page">
```

Comment:
I just realized that this file is in the `04-templates/example-page` directory. I think it should be in the `05-pages` directory (and not in a sub-directory) since it contains actual data, and not "replacement blocks".